packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -2686,7 +2686,198 @@ describe('NetworkController', () => {
+                }),
+              },
+              infuraProjectId: 'some-infura-project-id',
+            },
+            async ({ controller, messenger }) => {
+              const fakeProviders = [
+                buildFakeProvider([
+                  {
+                    request: {
+                      method: 'eth_getBlockByNumber',
+                    },
+                    response: SUCCESSFUL_ETH_GET_BLOCK_BY_NUMBER_RESPONSE,
+                    beforeCompleting: async () => {
+                      messenger.clearSubscriptions();
+                      await controller.destroy();
+                    },
+                  },
+                ]),
+              ];
+              const fakeNetworkClients = [buildFakeClient(fakeProviders[0])];
+              mockCreateNetworkClient()
+                .calledWith({
+                  chainId: toHex(1337),
+                  rpcUrl: 'https://mock-rpc-url',
+                  type: NetworkClientType.Custom,
+                  ticker: 'TEST',
+                })
+                .mockReturnValue(fakeNetworkClients[0]);
+
+              expect(controller.initializeProvider()).toBeFulfilled();
+            },
+          );
+        });
+
+        it('lookupNetwork keeps the state intact', async () => {
+          await withController(
+            {
+              state: {
+                providerConfig: buildProviderConfig({
+                  type: NetworkType.rpc,
+                  chainId: toHex(1337),
+                  rpcUrl: 'https://mock-rpc-url',
+                }),
+              },
+              infuraProjectId: 'some-infura-project-id',
+            },
+            async ({ controller, messenger }) => {
+              const fakeProviders = [
+                buildFakeProvider([
+                  {
+                    request: {
+                      method: 'eth_getBlockByNumber',
+                    },
+                    response: SUCCESSFUL_ETH_GET_BLOCK_BY_NUMBER_RESPONSE,
+                    beforeCompleting: async () => {
+                      messenger.clearSubscriptions();
+                      await controller.destroy();
+                    },
+                  },
+                ]),
+              ];
+              const fakeNetworkClients = [buildFakeClient(fakeProviders[0])];
+              mockCreateNetworkClient()
+                .calledWith({
+                  chainId: toHex(1337),
+                  rpcUrl: 'https://mock-rpc-url',
+                  type: NetworkClientType.Custom,
+                  ticker: 'TEST',
+                })
+                .mockReturnValue(fakeNetworkClients[0]);
+
+              await controller.initializeProvider();
+
+              expect(
+                controller.state.networksMetadata['https://mock-rpc-url']
+                  .status,
+              ).toBe(NetworkStatus.Unknown);
+            },
+          );
+        });
 
+        it.only('does not update network metadata property', async () => {
```

Comment:
Again, it does update this property, but it just doesn't fill it in. So perhaps:

``` suggestion
        it.only('does not capture whether the network supports EIP-1559', async () => {